src/game_constants.h

```diff
@@ -118,7 +118,7 @@ constexpr time_duration CORPSE_ROT_TIME = 24_hours;
 #define MAX_SKILL 10
 
 // Maximum (effective) level for a stat.
-#define MAX_STAT 20
+#define MAX_STAT 14
```

Comment:
Is this intended to lower max cap on _all stats_, or just strength per the PR title?

Reply:
All stats, whoops! Little bit of tunnel vision there.